From 58827e709eb7a2e0899260893a5c9c58eb0c5db1 Mon Sep 17 00:00:00 2001
From: Jonas Gorski <jogo@openwrt.org>
Date: Tue, 16 Jul 2013 14:04:40 +0200
Subject: [PATCH V2 06/13] MIPS: BMIPS: select CPU_HAS_PREFETCH

As they are MIPS32 CPUs they do support the prefetch opcode.

Signed-off-by: Jonas Gorski <jogo@openwrt.org>
---
 arch/mips/Kconfig | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -1467,6 +1467,7 @@ config CPU_BMIPS
 	select SWAP_IO_SPACE
 	select WEAK_ORDERING
 	select CPU_SUPPORTS_HIGHMEM
+	select CPU_HAS_PREFETCH
 	help
 	  Support for BMIPS3300/4350/4380 and BMIPS5000 processors.
 
